code/game/atoms/atoms.dm

```diff
@@ -824,18 +824,18 @@
 		var/image/I = generate_speech_bubble(src, "[bubble_icon][say_test(message)]", FLY_LAYER)
 		I.appearance_flags = APPEARANCE_UI_IGNORE_ALPHA
 		INVOKE_ASYNC(GLOBAL_PROC, /.proc/flick_overlay, I, speech_bubble_hearers, 30)
-		INVOKE_ASYNC(src, /atom/movable/proc/animate_chat, message, null, FALSE, speech_bubble_hearers, 30)
+		send_runechat_to_atom_movable(message = message, speaking = null, small = FALSE, show_to = speech_bubble_hearers, duration = 3 SECONDS)
 
 /atom/proc/say_overhead(var/message, whispering, message_range = 7, var/datum/language/speaking = null, var/list/passed_hearing_list)
 	var/list/speech_bubble_hearers = list()
-	var/italics
-	if(whispering)
-		italics = TRUE
 	for(var/mob/M in get_mobs_in_view(message_range, src))
 		if(M.client)
 			speech_bubble_hearers += M.client
 	if(length(speech_bubble_hearers))
-		INVOKE_ASYNC(src, /atom/movable/proc/animate_chat, message, speaking, italics, speech_bubble_hearers, 30)
+		send_runechat_to_atom_movable(message = message, speaking = speaking, small = whispering, show_to = speech_bubble_hearers, duration = 3 SECONDS)
+
+/atom/proc/send_runechat_to_atom_movable(message, speaking, small, list/show_to, duration = 3 SECONDS)
+	INVOKE_ASYNC(src, TYPE_PROC_REF(/atom/movable, animate_chat), message, speaking, small, show_to, duration)
```

Comment:
y not just make `animate_chat()` waitfor = FALSE and avoid the 2 extra proc calls